registry/storage/driver/oss/oss_test.go

```diff
@@ -128,13 +128,19 @@ func TestEmptyRootList(t *testing.T) {
 	defer rootedDriver.Delete(ctx, filename)
 
 	keys, err := emptyRootDriver.List(ctx, "/")
+	if err != nil {
+		t.Fatalf("unexpected error listing empty root content: %v", err)
+	}
```

Comment:
These area also missing on `main`; https://github.com/distribution/distribution/blob/983358f8e2509bf8ae196a8e135180a2c5b11264/registry/storage/driver/oss/oss_test.go#L130

Reply:
Done in https://github.com/distribution/distribution/pull/3950